packages/core/src/jsx.js

```diff
@@ -158,7 +158,7 @@ export const jsx: typeof React.createElement = function(
       )
       if (!match) {
         // safari and firefox
-        match = error.stack.match(/^.*\n([A-Z][A-Za-z$]+)@/)
+        match = error.stack.match(/.*\n([A-Z][A-Za-z$]+)@/)
```

Comment:
Example stack trace which was failing:
```
jsx@http://localhost:3000/_next/static/development/pages/theme_ui.js?ts=1578745166666:440:17
jsx$$1@http://localhost:3000/_next/static/development/pages/theme_ui.js?ts=1578745166666:2777:60
TestThemeUI@http://localhost:3000/_next/static/development/pages/theme_ui.js?ts=1578745166666:23:62
renderWithHooks@http://localhost:3000/_next/static/development/dll/dll_d6a88dbe3071bd165157.js?ts=1578745166666:16511:27
mountIndeterminateComponent@http://localhost:3000/_next/static/development/dll/dll_d6a88dbe3071bd165157.js?ts=1578745166666:19045:13
```

The Chrome one is not matching with `^` at the beginning, so this should be OK here too.